### ISSUE TITLE
FreTS.py

### ISSUE THREAD

**Issue by tongheng380** [aikunyi/FreTS#6]
if self.channel_independence == '1':
            x = self.MLP_channel(x, B, N, T)
self.channel_independence == '1' has a problem.

**Comment by breadacker** [comment]
yeah, channel independence doesnt work as expected. can someone clarify it?

**Comment by lidexf** [comment]
as ablation study in the paper, when the forecasting task is long term, channel dependence isn't work well. maybe just consider temporal learner in long-term forecasting would better that consider two dependence.

> if self.channel_independence == '1': x = self.MLP_channel(x, B, N, T) self.channel_independence == '1' has a problem.
